libtsuba/src/RDG.cpp

```
@@ -372,12 +376,13 @@ DoLoadPartialTable(const std::string& expected_name,
 
   auto combine_result = out->CombineChunks(arrow::default_memory_pool());
   if (!combine_result.ok()) {
-    GALOIS_LOG_DEBUG("arrow error: {}", combine_result.status());
-    return tsuba::ErrorCode::ArrowError;
+    GALOIS_LOG_DEBUG("{}, arrow error: {}", tsuba::Comm()->ID,
```

Better to warn as this is not returned as an error anymore. Silent errors during release builds will be confusing.

---

Also, it would useful to report the name and/or type of the property for which this error occurs.

---

The analytics applications do depend on chunks being combined into one. My preference would be to keep that invariant for all users of properties (querying, analytics) and perhaps figure out the root cause of the issue with LDBC 100.

If we do go forward with this change (perhaps to ease deployment during testing), can you file a ticket to revisit this change?

I think reporting the column in question in the debug or warn message would be an improvement regardless.